app/domain/authentication/authn_jwt/signing_key/create_jwks_from_http_response.rb

```diff
@@ -25,15 +26,17 @@ def validate_response_has_a_body
           raise Errors::Authentication::AuthnJwt::InvalidHttpResponseFormat unless @http_response.respond_to?(:body)
         end
 
-        def create_jwks_from_http_respnse
+        def create_jwks_from_http_response
           @logger.debug(LogMessages::Authentication::AuthnJwt::CreatingJwksFromHttpResponse.new)
+          parse_jwks_response(response_body, encoded_body)
```

Comment:
@tzheleznyak after your changes i think we no longer need to send those parameters to the function

Reply:
good point